database/pg/context.go

```diff
@@ -53,13 +53,12 @@ var dbKey key
 // associated database.
 //
 // Note: if a transaction is already pending in the passed-in context,
-// this function does not create a new one but returns the existing
-// one.
+// this function panics.
 func Begin(ctx context.Context) (Committer, context.Context, error) {
```

Comment:
> Are we at the point yet where we can delete this function entirely? It is one of the "context convenience functions".

Good call. There are there call sites, all in `core/account/utxodb`. I'll remove it in this PR.